scripts/deploy/medic-deploy.md

```diff
@@ -27,15 +27,18 @@ As each deployment needs its own configuration directory, DNS entry and deployme
 | MoH Uganda | `users-chis-ug`| `users-chis-ug.yaml` | `users-chis-ug-cht-user-management` | users-chis-ug.app.medicmobile.org | 
 | MoH Mali CIV | `users-chis-civ`| `users-chis-civ.yaml` | `users-chis-civ-cht-user-management` | users-chis-civ.app.medicmobile.org | 
 
-### `helm` commands
+### Prepare a Deployment
 
-The `helm upgrade` and `helm install` commands should be run in the `./scripts/deploy` directory in this repo.
+To prepare a deployment, use the following steps to ensure your environment is properly configured.
+The `helm install` and `helm upgrade` commands should be run in the `./scripts/deploy` directory in this repo.
 
 1. Check the image is [published](https://github.com/medic/cht-user-management/tree/main#publishing-new-docker-images) to [ECR](https://gallery.ecr.aws/medic/cht-user-management) 
-2. Commit a values YAML file in [values folder](https://github.com/medic/cht-user-management/blob/main/scripts/deploy/values/) in `main` branch
-3. Switch to the `deploy` branch and ensure the same values file from the prior step is in the [values folder](https://github.com/medic/cht-user-management/blob/deploy/scripts/deploy/values/) in `deploy` branch
-4. Edit the version in the `tag:` as the version you want deploy in the `helm` command below. Ensure this version in the `tag:` is commmited to `deploy` branch
-5. run either `helm install...` or `helm deploy...` per the full commands below. 
```

Comment:
Previously we wanted to use the `deploy` branch so that we didn't need to do a whole PR every time we wanted to push.  I also think this might result in extra images being built and pushed to ECR? [Slack thread](https://medic.slack.com/archives/C066J5C8Z88/p1711127804146269?thread_ts=1711127465.822439&cid=C066J5C8Z88):

>  For deployments, I've been pushing to [this deploy branch](https://github.com/medic/cht-user-management/pull/72) which is probably lazy but I also don't want to make a new PR every time I push. And when I merge it, publishes duplicate images.

This changes that - but maybe it's OK because we want to not use `deploy` branch and over all are changing things less often?

Reply:
> Previously we wanted to use the deploy branch so that we didn't need to do a whole PR every time we wanted to push....

Oh yes, I got it and I think we still don't want to make a whole PR for every push, neither duplicated images published... :) 

So maybe I should bring back the `deploy` branch use ? I am not sure I know the best thing to do here 😅

Reply:
I'd say let's revisit it in the `#cht-user-management` and we can discuss async.

Reply:
Sure